src/mono/nuget/Microsoft.NET.Sdk.WebAssembly.Pack/build/Microsoft.NET.Sdk.WebAssembly.Browser.targets

```diff
@@ -168,16 +168,24 @@ Copyright (c) .NET Foundation. All rights reserved.
     <Warning Condition="'$(BlazorWebAssemblyLoadAllGlobalizationData)' == 'true' AND '$(BlazorIcuDataFileName)' != ''" Text="%24(BlazorIcuDataFileName) has no effect when %24(BlazorWebAssemblyLoadAllGlobalizationData) is set to true." />
     <Warning Condition="'$(InvariantGlobalization)' == 'true' AND '$(HybridGlobalization)' == 'true'" Text="%24(HybridGlobalization) has no effect when %24(InvariantGlobalization) is set to true." />
     <Warning Condition="'$(BlazorIcuDataFileName)' != '' AND '$(HybridGlobalization)' == 'true'" Text="%24(HybridGlobalization) has no effect when %24(BlazorIcuDataFileName) is set." />
+    <FindInList CaseSensitive="true" List="@(ReferenceCopyLocalPaths)" MatchFileNameOnly="true" ItemSpecToFind="$(BlazorIcuDataFileName)">
```

Comment:
`ReferenceCopyLocalPaths` includes files other than the runtime pack too. Instead, use https://github.com/radical/runtime/blob/dccad58e258180650696613d872288efe413fdc6/src/mono/wasm/build/WasmApp.targets#L268-L268 to get the runtime pack, and then check for the file under that.

Reply:
This value is empty when we enter `_ResolveGlobalizationConfiguration` task. Checking the binlog we can see that there is no property that would hold this value, beyond specific items that have it as a part of their path.

<img width="664" alt="image" src="https://github.com/dotnet/runtime/assets/32700855/0a8c14bd-4632-4021-b351-76b3e5cac7fd">

Reply:
Because of that I am planning to preserve the original approach. To break it, a file with the same name would need to be added to `ReferenceCopyLocalPaths`.

Reply:
Can you please share the binlog?

Reply:
oh, just saw the `edit:` :)